README.md

```diff
@@ -1 +1 @@
-testing semantic versioning 5.0.0
```

Comment:
The code changes appear to be updating the version number from 5.0.0 to 5.2.0 for testing semantic versioning. This change seems reasonable and does not raise any concerns.